test/powershell/Provider/FileSystemProvider.Tests.ps1

```diff
@@ -0,0 +1,49 @@
+﻿Describe "SplitPath Tests (Windows Only)" -tags "CI" {
+
+    BeforeAll {
+        $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+        if ( ! $IsWindows ) {
+            $PSDefaultParameterValues["it:skip"] = $true
+        }
+    }
+    AfterAll {
+        $global:PSDefaultParameterValues = $originalDefaultParameterValues
+    }
+    
+    It 'Splits a multilevel path' {
+        Split-Path -Path 'C:\Temp\folder1' | Should be 'C:\temp'
+    }
+
+    It 'Returns a multilevel path child' {
+        Split-Path -Path 'C:\Temp\folder1' -Leaf | Should be 'folder1'
+    }
+   
+    It 'Splits a single level path' {
+        Split-Path -Path 'C:\Temp' | Should be 'C:\'
```

Comment:
For repeated tests please use [-Testcase ](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#parameter-generation)